src/systemd/cockpit.service.in

```diff
@@ -3,19 +3,16 @@ Description=Cockpit Web Service
 Documentation=man:cockpit-ws(8)
 Requires=cockpit.socket
 Requires=cockpit-wsinstance-http.socket cockpit-wsinstance-https-factory.socket
-# ensure our DynamicUser exists
-Requires=cockpit-ws-user.service cockpit-wsinstance-socket-user.service
-After=cockpit-ws-user.service cockpit-wsinstance-socket-user.service
 # we need to start after the sockets so that we can instantly forward incoming requests
 After=cockpit-wsinstance-http.socket cockpit-wsinstance-https-factory.socket
 
 [Service]
 RuntimeDirectory=cockpit/tls
 ExecStartPre=+@libexecdir@/cockpit-certificate-ensure --for-cockpit-tls
 ExecStart=@libexecdir@/cockpit-tls
-User=cockpit-ws
-Group=cockpit-ws
-SupplementaryGroups=cockpit-wsinstance-socket
+DynamicUser=yes
+User=cockpit-systemd-service  # otherwise systemd uses 'cockpit' even if it exists as a normal user account
```

Comment:
Haha:

> Accepting user/group name 'cockpit-systemd-service  # otherwise systemd uses 'cockpit' even if it exists as a normal user account', which does not match strict user/group name rules.

The comment should better go on its own line.

Also, the commit message says "cockpit-systemd-unit" which I find a bit too strange. I agree to not using "cockpit" here (and we most definitively must not use "cockpit-ws"). I can live with "cockpit-systemd-service"; "cockpit-service" or "cockpit-systemd" would be fine as well.

Reply:
What I wrote here is right.  The commit message is wrong.  I'm okay with `cockpit-service` if you strongly prefer it.  I first tried `cockpit.service` but ran into trouble with something or other not liking the dot.

Reply:
I kept c-systemd-service for now and just fixed the comment. Local image-prepare is happy now (tested f40 and d-testing).

It's nice to be able to change this stuff now, without committing anything to disk!